cachi2/core/package_managers/gomod.py

```diff
@@ -472,6 +472,7 @@ def _resolve_gomod(
         "GOCACHE": tmp_dir,
         "PATH": os.environ.get("PATH", ""),
         "GOMODCACHE": "{}/pkg/mod".format(tmp_dir),
+        "GOSUMDB": "sum.golang.org",
```

Comment:
Let's file a follow-up story to figure out how and where the users / the RHTAP maintainers will set GOPRIVATE (https://go.dev/ref/mod#environment-variables). Otherwise, this will break projects with private go deps